packages/core/src/Provider/Provider.ts

```diff
@@ -1,5 +1,17 @@
+import { Percent } from '../Cardano';
+import { Tip } from '@cardano-ogmios/schema';
+
 export type HealthCheckResponse = {
   ok: boolean;
+  localNode?: {
```

Comment:
Shouldn't this only be on `CardanoNodeHealthCheckResponse`, like
```ts
interface HealthCheckResponse { ok: boolean }
interface CardanoNodeHealthCheckResponse extends HealthCheckResponse {
  localNode: { ... }
}
```

Reply:
I followed the task requirements to extend the type as follows:
![Screenshot 2022-08-16 at 15 32 30](https://user-images.githubusercontent.com/10476819/184880506-81adcb24-ad03-4970-a7a7-e1540df083a4.png)

Reply:
I see, ok let's leave it as is then. There is value for all services to include this information.